trade-finance-manager-api/src/v1/__mocks__/amendmentVariables.js

```diff
@@ -0,0 +1,180 @@
+  ...approvedWithoutConditionsBothAmendments,
+  amendment: {
+    ukefDecision: {
+      value: CONSTANTS.DEALS.AMENDMENT_UW_DECISION.APPROVED_WITH_CONDITIONS,
+      coverEndDate: CONSTANTS.DEALS.AMENDMENT_UW_DECISION.APPROVED_WITH_CONDITIONS,
+      managersDecisionEmail: true,
+      conditions: 'ABCD',
+    },
+    ukefFacilityId: '4444',
+  },
+};
+
+const approvedWithConditionsDeclined = {
+  ...approvedWithoutConditionsBothAmendments,
+  amendment: {
+    ukefDecision: {
+      value: CONSTANTS.DEALS.AMENDMENT_UW_DECISION.APPROVED_WITH_CONDITIONS,
+      coverEndDate: CONSTANTS.DEALS.AMENDMENT_UW_DECISION.DECLINED,
+      managersDecisionEmail: true,
+      conditions: 'ABCD',
+      declined: 'EFGH',
+    },
+    ukefFacilityId: '4444',
+  },
+};
+
+const approvedWithConditionsDeclinedSwapped = {
+  ...approvedWithoutConditionsBothAmendments,
+  amendment: {
+    ukefDecision: {
+      value: CONSTANTS.DEALS.AMENDMENT_UW_DECISION.DECLINED,
+      coverEndDate: CONSTANTS.DEALS.AMENDMENT_UW_DECISION.APPROVED_WITH_CONDITIONS,
+      managersDecisionEmail: true,
+      conditions: 'ABCD',
+      declined: 'EFGH',
+    },
+    ukefFacilityId: '4444',
+  },
+};
+
+const approvedWithoutConditionsDeclined = {
+  ...approvedWithoutConditionsBothAmendments,
+  amendment: {
+    ukefDecision: {
+      value: CONSTANTS.DEALS.AMENDMENT_UW_DECISION.APPROVED_WITHOUT_CONDITIONS,
+      coverEndDate: CONSTANTS.DEALS.AMENDMENT_UW_DECISION.DECLINED,
+      managersDecisionEmail: true,
+      declined: 'EFGH',
+    },
+    ukefFacilityId: '4444',
+  },
+};
+
+const approvedWithoutConditionsDeclinedSwapped = {
+  ...approvedWithoutConditionsBothAmendments,
+  amendment: {
+    ukefDecision: {
+      value: CONSTANTS.DEALS.AMENDMENT_UW_DECISION.DECLINED,
+      coverEndDate: CONSTANTS.DEALS.AMENDMENT_UW_DECISION.APPROVED_WITHOUT_CONDITIONS,
+      managersDecisionEmail: true,
+      declined: 'EFGH',
+    },
+    ukefFacilityId: '4444',
+  },
+};
+
+const declinedBothAmendments = {
+  ...approvedWithoutConditionsBothAmendments,
+  amendment: {
+    ukefDecision: {
+      value: CONSTANTS.DEALS.AMENDMENT_UW_DECISION.DECLINED,
+      coverEndDate: CONSTANTS.DEALS.AMENDMENT_UW_DECISION.DECLINED,
+      managersDecisionEmail: true,
+      declined: 'EFGH',
+    },
+    ukefFacilityId: '4444',
+  },
+};
+
+const declinedOneAmendment = {
+  ...approvedWithoutConditionsBothAmendments,
+  amendment: {
+    ukefDecision: {
+      value: CONSTANTS.DEALS.AMENDMENT_UW_DECISION.DECLINED,
+      managersDecisionEmail: true,
+      declined: 'EFGH',
+    },
+    ukefFacilityId: '4444',
+  },
+};
+
+const wrongAmendments = {
+  ...approvedWithoutConditionsBothAmendments,
+  amendment: {
+    ukefDecision: {
+      value: 'test',
+      coverEndDate: 'test',
+      managersDecisionEmail: true,
+      declined: 'EFGH',
+    },
+    ukefFacilityId: '4444',
+  },
+};
+
+module.exports = { approvedWithoutConditionsBothAmendments,
```

Comment:
new line for `approvedWithoutConditionsBothAmendments`

Reply:
done